ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3058,3 +3058,93 @@ def test_tensorflow_Elu(
         features=x[0],
         name=name,
     )
+
+
+@st.composite
+def _conv2d_helper(draw):
+    input_dtype, \
+        x, \
+        filters, \
+        strides, \
+        dilations, \
+        data_format, \
+        padding = draw(_x_and_filters(
+        dtypes=helpers.get_dtypes("float", full=False),
+        data_format=st.sampled_from(["NHWC", "NCHW"]),
+        padding=st.sampled_from(["SAME", "VALID", "EXPLICIT"]),
+        type="2d",
+        dilation_min=1,
+        dilation_max=1,
+    ))
+    if padding == "EXPLICIT":
+        explicit_paddings = padding
+        pad_top = draw(st.integers(min_value=0, max_value=3))
```

Comment:
You don't really need to set `explicit_paddings = padding`. You set the value for `explicit_paddings` later on in your code.

Reply:
if we are making the explicit_padding = padding, then why are we returning the explicit_padding variable as well